src/app/_utils/convertMarkdownToEventData.ts

```diff
@@ -11,6 +11,14 @@ export function convertMarkdownToEventData(data: Record<string, any>) {
     location: data.location,
     externalLink: data['external-link'],
     lumaCalendarLink: data['luma-calendar-link'],
+    lumaEventsSection:
+      data['luma-events-section']?.title &&
+      data['luma-events-section']['embed-link']
+        ? {
```

Comment:
Does this mean that the section will only show if it has both fields filled out? If they put the embedded link, but not the title, it would return undefined?

Reply:
Yes! Added a hint to make this clear.